dev_guide/quota.adoc

```diff
@@ -11,59 +11,99 @@ toc::[]
 
 == Overview
 
-A *ResourceQuota* object enumerates hard resource usage limits per project. It
-limits the total number of a particular type of object that may be created in
-a project, and the total amount of
-link:compute_resources.html[compute resources]
+A *ResourceQuota* defines constraints that limit aggregrate resource consumption
+per project.  It can limit the quantity of objects that can be created in a project
+by type, as well as the total amount of link:compute_resources.html[compute resources]
 that may be consumed by resources in that project.
 
-== Usage Limits
+== Resources managed by quota
 
-The following describes the set of limits that may be enforced by a
-*ResourceQuota*.
+The following describes the set of resources that may be managed by a *ResourceQuota*.
 
-.Usage limits
+.Resources managed by quota
```

Comment:
There's also `openshift.io/images` (https://github.com/openshift/origin/pull/7930). But it hasn't been merged yet. I can make a follow-up.